tests/unit/test_api_delete.py

```diff
@@ -0,0 +1,12 @@
+"""Unit tests for DELETE /ctms/{primar_email}"""
+
+
+def test_delete_contact_by_primary_email_not_found(client):
+    resp = client.delete("/ctms/foo@bar.com")
+    assert resp.status_code == 404
+
+
+def test_delete_contact_by_primary_email(client, minimal_contact):
+    primary_email = minimal_contact.email.primary_email
+    resp = client.delete(f"/ctms/{primary_email}")
+    assert resp.status_code == 200
```

Comment:
Could you check that delete actually deleted something? For example, that a subsequent get or delete would return 404?

Reply:
Also, using `maximal_contact` would allow to test more of the model deletions (since it has more related objects)

Reply:
done :+1: